plugins/proxy/functions

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
+source "$PLUGIN_AVAILABLE_PATH/config/functions"
+
+is_app_proxy_enabled() {
+  local desc="return true if proxy is enabled; otherwise return false"
+  local APP="$1"; verify_app_name "$APP"
+  local APP_PROXY_ENABLED=true
+
+  local DOKKU_DISABLE_PROXY=$(config_get $APP DOKKU_DISABLE_PROXY)
+  if [[ -n "$DOKKU_DISABLE_PROXY" ]]; then
+    local APP_PROXY_ENABLED=false
+  fi
+  echo $APP_PROXY_ENABLED
+}
+
+get_app_proxy_type() {
```

Comment:
This functionality isn't fully fleshed out yet as we don't have another proxy yet.